data/json/items/gunmod/rail.json

```diff
@@ -54,5 +54,23 @@
     "min_skills": [ [ "weapon", 2 ], [ "gun", 1 ] ],
     "aim_cost": 2,
     "sight_dispersion": 360
+  },
+  {
+    "id": "stabilizer",
+    "type": "GUNMOD",
+    "name": "gyroscopic stabilizer",
+    "description": "An advanced unit that straps onto the side of your firearm and reduces vibration, greatly reducing recoil and increasing accuracy.",
+    "weight": 286,
+    "volume": 3,
+    "integral_volume": 2,
+    "price": 68000,
+    "material": [ "aluminum", "plastic" ],
+    "symbol": ":",
+    "color": "blue",
+    "location": "rail",
+    "mod_targets": [ "smg", "rifle", "crossbow" ],
+    "min_skills": [ [ "weapon", 6 ] ],
+    "dispersion_modifier": -30,
+    "handling_modifier": 6
```

Comment:
Probably. For the time being it at least now competes with another meaningful mod slot - before it was just a free bonus.